ios/RCCNavigationController.m

```diff
@@ -161,6 +162,19 @@ - (void)performAction:(NSString*)performAction actionParams:(NSDictionary*)actio
       [self setButtons:rightButtons viewController:viewController side:@"right" animated:NO];
     }
     
+    NSString *previewViewID = passProps[@"previewViewID"];
+    if (previewViewID) {
+      RCCViewController *topViewController = ((RCCViewController*)self.topViewController);
```

Comment:
This seems a little scary. Do we have a way to guarantee that this view controller will be a `RCCViewController`? Perhaps we should wrap this cast with a check to see if the view controller responds to the specific selector that we're calling on it (`@selector(setPreviewController:)`).

Reply:
Yes you're right, in the `if(keepStyleAcrossPushBool)` statement above it is  actually doing an `if ([self.topViewController isKindOfClass:[RCCViewController class]])`.

I will incorporate this, which one would you prefer?

Reply:
Either `isKindOfClass` or `respondsToSelector` is fine with me!